src/include/processor/operator/aggregate/hash_aggregate.h

```diff
@@ -114,10 +114,8 @@ class HashAggregateSharedState final : public BaseAggregateSharedState,
 };
 
 struct HashAggregateLocalState {
-    std::vector<common::ValueVector*> flatKeyVectors;
-    std::vector<common::ValueVector*> unFlatKeyVectors;
+    std::vector<common::ValueVector*> keyVectors;
```

Comment:
Why remove the leading state? leading state is a static information which can be computed at compile time, so we don't have to compute it over again during execution.